### DISCUSSION TITLE
Migration of education benefits to another account

### DISCUSSION THREAD

**Discussion by Marin-Kitagawa** [community/community#40977] [GitHub Education]
I want to migrate all the education benefits from one account to another. Is this possible? If yes, how to do this?

Is merging of two accounts, say A and B to A (with username retained as A) possible? Will all the education benefits of B be transferred to A as well? I have seen the feature of choosing a successor of the account but I am not sure of the functionality.

**Comment by samus-aran** [comment]
Hi @Marin-Kitagawa :wave:

It's not possible to transfer Global Campus benefits over from one account to another. 

EDIT: You are able to merge your accounts, but the account with the benefits would be the main account - you can add your non academic email from the previous account to this account as well as rename it once merged. For more details checkout out the [merging your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation. 

To learn more about adding an successor to an account [check out our documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/maintaining-ownership-continuity-of-your-personal-accounts-repositories). However this only applies to transferring user's repos not benefits.
